diff --git a/node_modules/react-dom/cjs/react-dom.development.js b/node_modules/react-dom/cjs/react-dom.development.js
index 13bea1e..23f8cd8 100644
--- a/node_modules/react-dom/cjs/react-dom.development.js
+++ b/node_modules/react-dom/cjs/react-dom.development.js
@@ -7408,7 +7408,13 @@ var normalizeHTML = void 0;
     // This tag is not present in non Electron environments such as JSDom which
     // is often used for testing purposes.
     // @see https://electronjs.org/docs/api/webview-tag
-    webview: true
+    webview: true,
+    // Custom
+    guitar: true,
+    layer: true,
+    string: true,
+    box: true,
+    boxunit: true,
   };
 
   validatePropertiesInDevelopment = function (type, props) {
